src/routes/userRoutes.js

```diff
@@ -43,4 +44,13 @@ router
   .route('/:id')
   .delete(authenticateUser, authorizePermissions('admin'), deleteSingleUser);
 
+// Route for rendering the upload page
+router.get('/:userId/uploadProfile', imgurController.renderUploadPage);
```

Comment:
A small nitpick. So while this works (after making the fixes I've noted in this PR, I was able to navigate to http://localhost:8000/api/v1/users/64ef5e90086e1c19b888c72c/uploadProfile and see the EJS page), it's a little awkward to go to a `/api` URL to view a page. All our `/api` routes should be for retruning API/JSON data.

What we've done with the other EJS files for now, is just have a standalone route in `expressServer`. So just for this `GET` route, I would suggest moving it to area of the code: https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/e8fb9a522bfdf8cdbb502c62e09e89dea6581dae/src/expressServer.js#L143-L150

Reply:
I have moved the route in expressServer 👍 , could you please check if it is correct? commit 27422acf5ecac14ca9f9edfe89abcb65694e369f